pkg/cpi/mockingUtils.go

```diff
@@ -295,33 +286,80 @@ func GetMockResponseByTestTypeAndMockFunctionName(mockFuntionName, testType stri
 							"code": "Bad Request",
 							"message": {
 							"@lang": "en",
-							"#text": "invalid request"
+							"#text": "Bad request"
 							}
 						}`))),
 			}
-			return &res, errors.New("Unable to get status of integration artifact, Response Status code:400")
+			return &res, errors.New("Unable to get integration artifact deploy status, Response Status code:400")
 		}
 
-	case "IntegrationArtifactDownloadCommandMockResponse":
+	case "GetIntegrationArtifactDeployErrorDetailsMockResponse":
 		if response == nil && error == nil {
 
 			res := http.Response{
-				StatusCode: 400,
+				StatusCode: 500,
 				Body: ioutil.NopCloser(bytes.NewReader([]byte(`{
-							"code": "Bad Request",
+							"code": "Internal Server Error",
 							"message": {
 							"@lang": "en",
-							"#text": "invalid request"
+							"#text": "Internal Processing Error"
 							}
 						}`))),
 			}
-			return &res, errors.New("Unable to download integration artifact, Response Status code:400")
+			return &res, errors.New("Unable to get integration artifact deploy error status, Response Status code:400")
 		}
 
 	}
 	return response, error
 }
 
+//NegtiveResForIntegrationDesigntimeArtifactUpdateMockResponse -Nagative Case http response body
+func NegtiveResForIntegrationDesigntimeArtifactUpdateMockResponse() (*http.Response, error) {
+	res := http.Response{
+		StatusCode: 400,
+		Body: ioutil.NopCloser(bytes.NewReader([]byte(`{
+					"code": "Bad Request",
+					"message": {
+					"@lang": "en",
+					"#text": "invalid request"
+					}
+				}`))),
+	}
+	return &res, errors.New("Unable to get status of integration artifact, Response Status code:400")
+}
+
+//NegtiveResForGetIntegrationDesigntimeArtifactMockResponse -Nagative Case http response body
+func NegtiveResForGetIntegrationDesigntimeArtifactMockResponse() (*http.Response, error) {
+
+	res := http.Response{
+		StatusCode: 400,
+		Body: ioutil.NopCloser(bytes.NewReader([]byte(`{
+					"code": "Bad Request",
+					"message": {
+					"@lang": "en",
+					"#text": "invalid request"
+					}
+				}`))),
+	}
+	return &res, errors.New("Unable to get status of integration artifact, Response Status code:400")
+}
+
+//NegtiveResForIntegrationArtifactDownloadCommandMockResponse -Nagative Case http response body
+func NegtiveResForIntegrationArtifactDownloadCommandMockResponse() (*http.Response, error) {
```

Comment:
I see no difference between these three functions:
- `NegtiveResForIntegrationDesigntimeArtifactUpdateMockResponse`
- `NegtiveResForGetIntegrationDesigntimeArtifactMockResponse`
- `NegtiveResForIntegrationArtifactDownloadCommandMockResponse`

Please refactor to one function.

Reply:
done